frontend/packages/client/src/components/Header.js

```diff
@@ -38,6 +38,17 @@ function Header(props) {
                     About
                   </span>
                 </NavLink>
+                <NavLink
+                  to={{
+                    pathname: "/community/create",
+                    state: { modal: true },
```

Comment:
```suggestion
```
this is being used in the about page to open as a modal here: https://github.com/DapperCollectives/CAST/blob/1a80cc71fb95913804dfc7f41338ecc9c9356e3b/frontend/packages/client/src/pages/About.js#L13 
it can be removed

Reply:
this one can be removed